SpatialGDK/Source/SpatialGDKTests/Private/RPCContainer/RPCContainerTest.cpp

```diff
@@ -0,0 +1,214 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "TestDefinitions.h"
+
+#include "ObjectDummy.h"
+#include "ObjectSpy.h"
+
+#include "Utils/RPCContainer.h"
+#include "Schema/RPCPayload.h"
+
+#include <Core.h>
```

Comment:
Is this needed?

Reply:
Yes, however, CoreMinimal might suffice if that's what you mean.

Reply:
If something smaller suffices then use that. I actually think CoreMinimal is itself still ludicrously large and should only be used if you don't know which specific headers to use.

Reply:
Updated.